pom.xml

```diff
@@ -27,15 +27,16 @@
     </developers>
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>2.440.3</jenkins.version>
+        <jenkins.version>2.452.4</jenkins.version>
         <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
+    <maven.compiler.release>17</maven.compiler.release>
```

Comment:
No! This should never be done this way in a plugin repo. If the dep is 2.479.x and the parent is 5.x then this is automatic.